FavoriteArtists/MKJArtistFetcher.m

```diff
@@ -0,0 +1,78 @@
+//
+//  MKJArtistFetcher.m
+//  FavoriteArtists
+//
+//  Created by Kenneth Jones on 12/3/20.
+//
+
+#import "MKJArtistFetcher.h"
+
+static NSString *const ArtistFetcherBaseURLString = @"https://www.theaudiodb.com/api/v1/json/1/search.php";
+
+@implementation MKJArtistFetcher
+
+- (instancetype)initWithArtists:(NSMutableArray<MKJArtist *> *)artists
+{
+    if (self = [super init]) {
+        _artists = [[NSMutableArray alloc] init];
+    }
+    return self;
+}
```

Comment:
So your project was not adding the artist on the table view, I believe because it isn't initializing the array that you are trying to hold your artist objects in. Try something like this...
```suggestion
- (instancetype)init {
    if (self = [super init]) {
        _artistsArray = [[NSMutableArray alloc] init];
    }
    return self;
}
```